aiosonic/connection.py

```diff
@@ -75,8 +76,8 @@ async def connect(
         ssl_context: SSLContext,
         http2: bool = False,
     ) -> None:
-        """Connet with timeout."""
-        self._verify = verify
+        """Connect with timeout."""
+        self.verify = verify
```

Comment:
```suggestion
        self._verify = verify
```

Reply:
Actually I think this variable is unused so I'm temped to remove it. Must be a remnant of leftover code.

Reply:
Or maybe it's a merge overlook. Probably the later